etc/deploy/aws/aws-doco-script.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash
+
+# Preq: Run aws config to configure auth context profile before running this script.
+
+set -xeou pipefail
+
+jq --version || echo "This script required jq, install jq from https://stedolan.github.io/jq/download/"
+psql --version || echo "This script required psql, install psql from https://www.postgresql.org/download/"
+
+GLOBAL_TAG="jose-testing-4"
```

Comment:
I recommend putting all of the variables which need to be changed up at the _very_ top, with a comment block indicating that fact.